Makefile

```diff
@@ -15,13 +23,13 @@ proto:
 # test runs all of the application tests locally.
 .PHONY: tests
 tests:
-	go clean -testcache
-	godotenv -f /etc/alpine-hodler/auth.env go test ./... -v
```

Comment:
Could we also remove these lines from the CI scrip: https://github.com/alpine-hodler/gidari/blob/main/scripts/run-ci-tests.sh#L5-L6